WalletWasabi/WabiSabi/Backend/Rounds/Arena.cs

```diff
@@ -127,15 +136,18 @@ private void StepOutputRegistrationPhase()
 					{
 						coinjoin.Inputs.Add(input);
 					}
+					round.LogInfo($"{coinjoin.Inputs.Count} inputs were added.");
 
 					// Add outputs:
 					foreach (var bob in round.Bobs)
 					{
 						coinjoin.Outputs.AddWithOptimize(bob.CalculateOutputAmount(round.FeeRate), bob.Script);
 					}
+					round.LogInfo($"{coinjoin.Outputs.Count} outputs were added.");
 
 					// Shuffle & sort:
 					// This is basically just decoration.
+					round.LogInfo("Shuffling and sorting inputs and outputs.");
```

Comment:
Does this help?

I mean how would one use this information? Or what is the goal behind this?

Reply:
Removed.